web/src/i18n/locales/en.yaml

```diff
@@ -67,7 +67,7 @@
         helpText: "Provide a brief, high level summary of the state's program, history, and future vision. Specific activities for the program will be explained in the program activities section of this update."
     hit: 
       title: "HIT Overview"
-      helpText: "Briefly summarize planned HIT funded activities contained within this APD, we'll ask for more specific details later."
+      helpText: "Briefly summarize planned HIT funded activities contained within this APD, ask for more specific details later."
```

Comment:
"ask for more specific details later" seems like it's not a complete sentence. Maybe something like "Briefly summarize planned HIT funded activities contained within this APD. Additional details will be provided later in the."

Reply:
That was an oversight I'll amend this.